### DISCUSSION TITLE
A/B testing anyone?

### DISCUSSION THREAD

**Discussion by mrpdean** [pimcore/pimcore#12503] [Q&A / Support]
Does anyone have any advice around how to go about A/B multiple versions of a page in Pimcore?

Can't seem to find any documentation around this.

Thanks

**Comment by NiklasBr** [comment]
See [VisitorInfo](https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Targeting_and_Personalization/Visitor_Info.html) docs, in there you could access that targeting data to serve personalised content. A/B testing is never mentioned, but it should be possible from there.
